orca-core/src/main/java/com/netflix/spinnaker/orca/clouddriver/utils/CloudProviderAware.java

```diff
@@ -17,9 +17,12 @@
 package com.netflix.spinnaker.orca.clouddriver.utils;
```

Comment:
nit: Surprised to see this here! This class, nor `RegionCollector` which uses this class, don't belong in `orca-core`, imo. I'd rather see these moved into `orca-clouddriver`.

Reply:
eh, is it really worth it? That means making sure all of these have a dependency on `orca-clouddriver`:

```
orca-mine
orca-queue
orca-queue-tck
orca-integrations-cloudfoundry
orca-bakery
```